plugins/item_previews/web_client/js/item_previews.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * The Item Preview widget shows a preview of items under a given folder.
+ * For now, the only supported item previews are image previews.
+ */
+
+girder.views.ItemPreviewWidget = girder.View.extend({
+
+    events: {
+        'click a.g-item-preview-link': function (event) {
+            // TODO: trigger a 'g:itemClicked' based on cid
+            // or other method rather than this lazy link proxy:
+            var name = $(event.currentTarget).attr('g-item-name');
+            $('.g-item-list a').filter(function (i, d) {
```

Comment:
Let's use `this.$` instead of the raw `$` so that we only apply to DOM elements underneath this widget.